gitbutler-ui/src/lib/components/CommitDialog.svelte

```diff
@@ -49,19 +48,19 @@
 	function getCommitMessageTitleAndDescription(commitMessage: string) {
 		// Split the commit message into title and description
 		// get the first line as title and the rest as description
-		const [summary, ...description] = commitMessage.split('\n');
+		const [summary, ...description] = commitMessage.trim().split(/\n+(.*)/s);
```

Comment:
You can remove the `...` now, making description a `string` rather than `string[]`.

Reply:
Ah, I see. That's why it's an array. I completely forgot about this one. Thanks!